test/indexer/test_hca_indexer.py

```diff
@@ -754,7 +754,7 @@ def test_metadata_field_exclusion(self):
         self.assertEqual(1, hits["hits"]["total"])
 
 
-class TestValidNotificationRequests(LocalAppTestCase):
+class TestValidNotificationRequests(WebServiceTestCase):
```

Comment:
Why?

Reply:
I think this was accidental. I'll revert and see if there aren't any problems.